src/main/kotlin/com/criptext/mail/utils/ui/TextDrawable.java

```diff
@@ -92,7 +92,7 @@ public void draw(Canvas canvas) {
         // draw text
         int width = this.width < 0 ? r.width() : this.width;
         int height = this.height < 0 ? r.height() : this.height;
-        int fontSize = this.fontSize < 0 ? (Math.min(width, height) / 2) : this.fontSize;
+        int fontSize = this.fontSize < 0 ? (Math.min(width, height) / 2) - 25 : this.fontSize;
```

Comment:
-25 may not be friendly for different screen sizes, if you need to do this use dp sizes.